api/python/cellxgene_census/tests/experimental/diffexp/memento/test_diff_expr.py

```diff
@@ -1,156 +1,97 @@
-import itertools
-from typing import Dict
+from os import path
+from typing import Any, Dict
 
+import numpy as np
 import pandas as pd
 import pytest
 
 from cellxgene_census.experimental.diffexp.memento import diff_expr
-from cellxgene_census.experimental.diffexp.memento.diff_expr import CUBE_LOGICAL_DIMS_OBS
 
 
-@pytest.fixture(scope="function")
-def estimators_df(dim_counts: Dict[str, int]) -> pd.DataFrame:
-    """Create a dummy estimators DataFrame, with all combinations of variables."""
-    columns = CUBE_LOGICAL_DIMS_OBS + ["feature_id"]
-    dim_counts_full = {var: 1 for var in columns}
-    dim_counts_full.update(dim_counts or {})
-    dim_values = {var: [f"{var}_{chr(ord('a') + i)}" for i in range(n)] for var, n in dim_counts_full.items()}
+class TestDiffExprRealDataset:
```

Comment:
why introduce a class? (doing so is non-idiomatic for pytest)

Reply:
pytest supports classes, yes, but I don't often see them used with pytest.  anyway, it's just a nit! :)